src/main/java/cholog/supp/api/study/controller/InvitationController.java

```diff
@@ -0,0 +1,24 @@
+package cholog.supp.api.study.controller;
+
+import cholog.supp.api.study.dto.request.InvitationRequest;
+import cholog.supp.api.study.service.InvitationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1")
+public class InvitationController {
+
+    private final InvitationService invitationService;
+
+    @GetMapping("/invitation")
+    public ResponseEntity<String> invitation(@RequestBody InvitationRequest invitationRequest) {
```

Comment:
1. 
```suggestion
    public ResponseEntity<String> getInvitationLink(@RequestBody InvitationRequest invitationRequest) {
```
2. 
```suggestion
    public ResponseEntity<String> getInvitationToken(@RequestBody InvitationRequest invitationRequest) {
```
여기도 앞에 `get`을 붙여서 1, 2 중 하나 어떨까??~~

Reply:
1번이 명확하고 좋은거같아!! 컨트롤러 메서드 이름이라 신경을 안썼네...

Reply:
좋아요!!

Reply:
나도 `get`을 붙이는게 의미 전달이 더 잘 되는 것 같아. 다만 URL 상에서는 초대장을 뜻하는 `Invitation`을 리소스로 명시하고 있어서 메서드 이름을 `getInvitation`으로 해도 충분하지 않을까하는 생각이 들었어. 링크와 관련된 로직은 `InvitationService`에서 확인가능한 부분이니까 컨트롤러 단에서는 `Link`라는 표현을 붙이지 않아도 될 것 같다는 생각!